pkg/controller/jaeger/jaeger_controller.go

```diff
@@ -237,50 +256,54 @@ func (r *ReconcileJaeger) apply(jaeger v1.Jaeger, str strategy.S) (v1.Jaeger, er
 	}
 
 	// storage dependencies have to be deployed after ES is ready
-	if err := r.handleDependencies(str); err != nil {
-		return jaeger, errors.Wrap(err, "failed to handler dependencies")
```

Comment:
Shouldn't the wrapped error be passed?

Reply:
Not sure about this one, though. I don't think it's adding much value here.

Reply:
Ok np - just wanted to point it out, in case was by mistake.